tests/performance/config/perf-driver/policies/test-continuous-n-apps.yml

```diff
@@ -49,7 +49,7 @@ policies:
     # Complete the policy when 10000 deployment completion events are received
     events:
       start: MarathonStartedEvent
-      end: MarathonDeploymentSuccessEvent:nth(10000,all) MarathonDeploymentFailedEvent:nth(10000,all)
+      end: MarathonDeploymentSuccessEvent:nth(10000,all) MarathonDeploymentFailedEvent:nth(10000,all) MarathonDeploymentRequestFailedEvent(10000,all)
```

Comment:
So we end if `all` the request fail?

Reply:
Nope, this is a grouping operator. It means that it will account overall `10000` events, either `MarathonDeploymentSuccessEvent`, or `MarathonDeploymentFailedEvent`, or `MarathonDeploymentRequestFailedEvent`.

This particular part is a bit boy-scouting, since I noticed that a job never finished because it was failing to place the request to marathon (aka `MarathonDeploymentRequestFailedEvent`) instead of marathon not being able to satisfy the request (aka `MarathonDeploymentFailedEvent`)